lib/cloudkeeper/entities/appliance.rb

```diff
@@ -4,7 +4,7 @@ class Appliance
       attr_accessor :identifier, :description, :mpuri, :title, :group, :ram, :core, :version, :architecture
       attr_accessor :operating_system, :image, :attributes, :vo, :expiration_date, :image_list_identifier
 
-      REJECTED_ATTRIBUTES = %i[vo expiration image_list_identifier].freeze
+      REJECTED_ATTRIBUTES = %i[vo image_list_identifier].freeze
 
       def initialize(identifier, mpuri, vo, expiration_date, image_list_identifier, title = '', description = '', group = '',
```

Comment:
At this point, wouldn't passing a hash be a better/cleaner option? There is a lot of arguments here.

Reply:
Yes, I know. Maybe I'll change it when/if moving to v2.x but I'll leave it as it is for now.